### PR TITLE
cmd/utils, internal/debug: fixes for debugging

### PR DESCRIPTION
**Pull request by fjl**

### CODE REVIEW

**Comment by robotally** [comment]
| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Fri Mar 11 23:44:55 UTC 2016_

**Comment by karalabe** [comment]
Don't we want to to enable full panics for spurious ones too? I.e. if something actually goes wrong?

**Comment by fjl** [comment]
The change in Go 1.6 is actually useful. Usually panics are related only to the goroutine that causes them (e.g. nil pointer, out of bounds) and the additional output is not helping. Sometimes people don't submit the whole log because it's too big.

**Comment by karalabe** [comment]
LGTM :+1:

File: internal/debug/flags.go [hunk 1]
```diff
@@ -55,6 +55,7 @@ var (
 	memprofilerateFlag = cli.IntFlag{
 		Name:  "memprofilerate",
 		Usage: "Turn on memory profiling with the given rate",
+		Value: runtime.MemProfileRate,
```
Comment (karalabe) [inline internal/debug/flags.go]: This doesn't automatically turn on profiling, right? Also shouldn't something similar be done for `blockprofilerate`?

Comment (fjl) [inline internal/debug/flags.go]: It basically restores the default value. Memory profiling is always "on" in most programs, but has low impact on performance.

Comment (fjl) [inline internal/debug/flags.go]: It's a different story for block profiling though, so its not on by default. But you can turn it on through the API, unlike MemProfileRate.

**Pull request merged**